hassio/src/addon-view/hassio-addon-logs.js

```diff
@@ -15,10 +15,72 @@ class HassioAddonLogs extends PolymerElement {
         }
         pre {
           overflow-x: auto;
+          white-space: pre-wrap;
+          overflow-wrap: break-word;
```

Comment:
This enables wrapping in the logs. I personally find it easier to read but I can also remove it again

<img width="954" alt="screen shot 2018-11-30 at 11 07 51" src="https://user-images.githubusercontent.com/6833237/49282783-4aab2c00-f490-11e8-86ca-11af1d9dcf24.png">
(before)
<img width="952" alt="screen shot 2018-11-30 at 11 07 39" src="https://user-images.githubusercontent.com/6833237/49282797-4ed74980-f490-11e8-9fdd-62832c6e66b3.png">
(after)